packages/mobile/src/web3/actions.ts

```diff
@@ -72,6 +79,13 @@ export const setAccount = (address: string): SetAccountAction => {
   }
 }
 
+export const setMtwAddress = (address: string | null): SetMtwAddressAction => {
+  return {
+    type: Actions.SET_MTW_ADDRESS,
+    address: address?.toLowerCase() ?? address,
```

Comment:
`address?.toLowerCase()` will return `null` if address is `null` so the `??` should be unnecessary. 
```suggestion
    address: address?.toLowerCase(),
```

Reply:
I think it actually returns `undefined` in the failure case so will leave it as it